products/sle/main.pm

```diff
@@ -646,6 +646,28 @@ sub prepare_target {
     }
 }
 
+sub is_baremetal_test {
+    return get_var('IBTESTS');
+}
+
+sub load_baremetal_tests {
```

Comment:
+1 to this function. I will adapt NFV tests to use this function so we don't repeat always load_boot_tests(), load_inst_tests() and load_reboot_tests().